### DISCUSSION TITLE
running lammps with error

### DISCUSSION THREAD

**Discussion by BaoHhhhhhan** [deepmodeling/deepmd-kit#681] [Q&A]
when I run lammps,I got the output below:
Not found: Op type not registered 'DescrptSeA' in binary running on node51-W580-G20. Make sure the Op and Kernel are registered in the binary running in this process. Note that if you are loading a saved graph which used ops from tf.contrib, accessing (e.g.) `tf.contrib.resampler` should be done before importing the graph, as contrib ops are lazily registered when the module is first accessed.

**Comment by njzjz** [comment]
Do you use the same version for training and LAMMPS?

**Reply by BaoHhhhhhan** [reply]
thanks!
I use pip installing deepmd python interface and train a model, then I run it with compiled LAMMPS and deepmd.

**Reply by njzjz** [reply]
Please provide the model for further debug.

**Reply by BaoHhhhhhan** [reply]
thanks very much！
I used setup.py installing python's interface,then I retrained a new model,which works fine with lammps.
